app/templates/admin/manage_categories.html

```diff
@@ -9,25 +9,25 @@
     <table>
         <thead>
             <tr>
-                <td>HTML Version</td>
-                <td>Human-readable Version</td>
+                <th>HTML Version</th>
+                <th>Human-readable Version</th>
                 <td></td>
                 <td></td>
             </tr>
         </thead>
         <tbody>
     {% for category in categories %}
             <tr>
-                <td>
+                <td data-th="HTML Version">
                     {{ category['category_html'] }}
                 </td>
-                <td>
+                <td data-th="Human-readable Version">
                     {{ category['category_human'] }}
                 </td>
-                <td>
+                <td data-th="Edit">
                     <a href="{{ url_for('View:edit_category', category_id = category['category_id']) }}" class="button">Edit</a>
                 </td>
-                <td>
+                <td data-th="Delete">
```

Comment:
tbh I think adding this data-th makes it look worse and takes up more room when shrunk down. 

See 

Current look 
![Screen Shot 2021-07-21 at 11 09 54 AM](https://user-images.githubusercontent.com/26125143/126522436-aed30fbc-4be3-4e0f-a869-6841523cb8bb.png)

Updated look
![Screen Shot 2021-07-21 at 11 08 59 AM](https://user-images.githubusercontent.com/26125143/126522322-12c3fbd6-7f3a-4c52-97d8-6e4fc5645b93.png)

Reply:
On the smallest screens the original view pushed everything offscreen and the navbar gets messed up a bit. Do you know of a way to make the original look good on a small screen?

Reply:
Alright, I see what you mean, probs just keep it like this then.